training/distributed_training/pytorch/model_parallel/gpt-j/train_gptj_smp_notebook.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>Optionally cite a recent paper or blog post on the release of GPT-J 6B?</p><p> </p><p>Grammar nit - please change all <code>Pytorch</code> -&gt; <code>PyTorch</code> , <code>Sagemaker</code> -&gt; <code>SageMaker</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>